sql-db/vertx-sql/src/test/java/io/quarkus/ts/vertx/sql/handlers/MssqlHandlerIT.java

```diff
@@ -14,7 +14,8 @@
 @QuarkusScenario
 public class MssqlHandlerIT extends CommonTestCases {
 
-    @SqlServerContainer
+    // TODO: use default (newer) image when https://github.com/quarkusio/quarkus/issues/46083 gets fixed
+    @SqlServerContainer(image = "mcr.microsoft.com/mssql/rhel/server:2022-CU10-rhel-9.1")
```

Comment:
Btw why not use `mcr.microsoft.com/mssql/rhel/server:2022-CU16-rhel-9.1` as from what I remeber I saw this issue appear around when CU17 was released (First time saw it here https://github.com/quarkus-qe/quarkus-test-suite/pull/2274)

Reply:
Done, never better, thanks.